fairscale/utils/containers.py

```diff
@@ -12,6 +12,9 @@
 """Useful functions to deal with tensor types with other python container types."""
 
 
+# FIXME: the semantics and definition of this are more flexible
```

Comment:
Will have to look at the code new version when you have it to have a concrete opinion on it. on the surface sounds nice.